cea/demand/calibration/bayesian_calibrator/calibration_main.py

```diff
@@ -38,95 +37,79 @@
             arguments_norm = min_max_scaler.fit_transform(arguments)
-            min = arguments_norm[0]
-            max = arguments_norm[1]
-            mu = arguments_norm[2]
-            stdv = arguments_norm[3]
-            if distribution == 'triangular':
-                loc = min
-                scale = max - min
-                c = (mu - min) / (max - min)
-                globals()['var' + str(i + 1)] = pymc3.Triangular('var' + str(i + 1), lower=loc, c=c, upper=max)
-            elif distribution == 'normal':
-                globals()['var' + str(i + 1)] = pymc3.Normal('var' + str(i + 1), mu=mu, sd=stdv)
-            else:  # assume it is uniform
-                globals()['var' + str(i + 1)] = pymc3.Uniform('var' + str(i + 1), lower=min, upper=max)
-
-            vars.append('var'+str(i+1))
-
-        # expected value of outcome
-        mu = pymc3.Deterministic('mu', calc_result_emulator_and_bias(var1, var2, var3, var4, var5))
-
-        # Likelihood (sampling distribution) of observations
-        sigma = pymc3.HalfNormal('sigma', sd=0.10)
-        observed = calc_observed_synthetic()
-        y_obs = pymc3.Normal('y_obs', mu=mu, sd=sigma, observed = 0)
-
-    if cea.demand.calibration.settings.generate_plots:
-        with basic_model:
-            # plot posteriors
-            trace = pymc3.backends.text.load(os.path.join(locator.get_calibration_folder()))
-            pymc3.traceplot(trace)
-            plt.show()
-    else:
+            globals()['var' + str(i + 1)] = pm.Triangular('var' + str(i + 1), lower=arguments_norm[0][0],
+                                                          c=arguments_norm[1][0],
+                                                          upper=arguments_norm[2][0])
+
+        # DECLARE OBJECTIVE FUNCTION
+        mu = pm.Deterministic('mu', predict_y(var1, var2, var3, var4, var5, var6))
+        sigma = pm.HalfNormal('sigma', 0.15)
+        # sigma = pm.Deterministic('sigma', predict_sigma(var1, var2, var3, var4, var5, var6))
+        y_obs = pm.Normal('y_obs', mu=mu, sd=sigma, observed=0)
+
+        # RUN MODEL, SAVE TO DISC AND PLOT RESULTS
         with basic_model:
-            step = pymc3.Metropolis()
-            trace = pymc3.sample(cea.demand.calibration.settings.max_iter_MCMC,
-                              tune=cea.demand.calibration.settings.burn_in, step=step)
-            pymc3.backends.text.dump(locator.get_calibration_folder(), trace)
-            pymc3.traceplot(trace)
+            # Running
+            step = pm.Metropolis()
+            trace = pm.sample(20000, tune=1000, step=step)
+            # Saving
+            df_trace = pm.trace_to_dataframe(trace)
+
+            #CREATE GRAPHS AND SAVE TO DISC
+            df_trace.to_csv(locator.get_calibration_posteriors(building_name, building_load))
+            pm.traceplot(trace)
+
+            columns = ["var1", "var2", "var3", "var4", "var5", "var6"]
+            sns.pairplot(df_trace[columns])
```

Comment:
sns is not being imported, so this line does not work

Reply:
fixed